enterprise/server/atime_updater/atime_updater.go

```diff
@@ -126,10 +129,15 @@ func (u *atimeUpdater) Enqueue(ctx context.Context, instanceName string, digests
 	u.mu.Unlock()
 
 	// Uniqueify the incoming digests. Note this wrecks the order of the
-	// requested updates. Too bad.
-	keys := map[digest.Key]struct{}{}
+	// requested updates. Too bad. Keep track of the counts for metrics below.
+	keys := map[digest.Key]int{}
 	for _, digestProto := range digests {
-		keys[digest.NewKey(digestProto)] = struct{}{}
+		key := digest.NewKey(digestProto)
+		if count, ok := keys[key]; ok {
```

Comment:
You can replace this code with `keys[key]++`

Reply:
If a key doesn't exist, you get the zero value back which is actually `0` in this case since the value is an int.

Reply:
Done